radarpipeline/io/ingestion.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from radarpipeline.io.reader import Reader
+from radarpipeline.project.sparkengine import SparkEngine
+
+from typing import Dict
+
+logger = logging.getLogger(__name__)
+
+
+class CustomDataReader():
```

Comment:
What is the distinction here between ingestion and reader?

Reply:
Nothing, I think ingestion is the filename. I tried to keep the function names which are exposed to the user simple and straight. That's why I named it that way.